### PR TITLE
Added Pull Requester Labeler

### PR DESCRIPTION
**Pull request by PseudoCodeNerd**
Added Pull Requester Labeler as part of a Task in Google Code In for Julia.

Sir, Please review.
@logankilpatrick

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Documentation:
```
Comment (logankilpatrick) [inline .github/labeler.yml]: For this file, the words above the file path correspond to Labels in the repo found here: https://github.com/giordano/PolynomialRoots.jl/labels. You need to use the existing labels or open an issue requesting the label to be created if it does not exist, like for "Test Suite Modification:". Though, for that one, it might be better to just have a simple "Testing" label.

Comment (PseudoCodeNerd) [reply .github/labeler.yml]: Ok. 
Will do!

Comment (PseudoCodeNerd) [reply .github/labeler.yml]: #17 Request for 2 new labels added. 

One has been pushed. 
Once I'm approved to add those as well, I'll do it.
I was trying to implement the "question" label too but even after learning RegEx to find literal characters, I had no luck. 

Please, can you complete my review with this one label. I assure you I'll add the new labels addressed in the issue which I've opened too.

Thanks!

File: .github/workflows/label.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: PR Labeler
+on: [pull_request]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+
+    steps:
```
Comment (logankilpatrick) [inline .github/workflows/label.yml]: Hey, Please revert the changes back to what you had before. The Label.yml file was correct and the format for the Labeler.yml file was also correct it just had the wrong labels!

Comment (logankilpatrick) [reply .github/workflows/label.yml]: You can leave those labels for now just make sure they are referenced in the issue you opened.

Comment (PseudoCodeNerd) [reply .github/workflows/label.yml]: Reverted https://github.com/giordano/PolynomialRoots.jl/pull/16/commits/cd9bc700e70d9bc4d73b0a0ba952f2cb1ec734bf .


Can I get approved now ?

Regards,
PseudoCodeNerd

**Comment by logankilpatrick** [comment]
Need's #17 before merging.

**Comment by logankilpatrick** [comment]
Why was this closed?

**Comment by PseudoCodeNerd** [comment]
Sorry sir. It was a mistake on my part.
I'll reopen it just now.
